pom.xml

```diff
@@ -70,6 +70,28 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>com.theoryinpractise</groupId>
+                <artifactId>googleformatter-maven-plugin</artifactId>
+                <version>1.7.5</version>
+                <executions>
+                    <execution>
+                        <id>reformat-sources</id>
+                        <configuration>
+                            <includeStale>false</includeStale>
+                            <style>AOSP</style>
```

Comment:
Leste meg litt opp på java code style uten å bli helt klok, men skjønte at AOSP var myntet på android-kode. Er det noen spesiell grunn til at AOSP brukes i stedet for feks GOOGLE code style?

Reply:
Den ser ut til å være meir konfigurerbar også. Full oversikt over muligheter https://help.eclipse.org/latest/topic/org.eclipse.jdt.doc.isv/reference/api/org/eclipse/jdt/core/formatter/DefaultCodeFormatterConstants.html